storage/ta_storage.h

```diff
@@ -10,10 +10,9 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
-
+#include "scylladb_Permanode.h"
```

Comment:
I am confused with the naming: uppercase "P" or lowercase "p"?

Reply:
I'd like to name it with lowercase "p".
Let me delete the old files with the same name "scylladb_permaonde.*" in #548 first.